app/models/grda_warehouse/warehouse_reports/report_definition.rb

```diff
@@ -1299,6 +1299,16 @@ def self.report_list # rubocop:disable Metrics/AbcSize
         }
       end
 
+      if RailsDrivers.loaded.include?(:all_neighbors_system_dashboard)
+        r_list['Performance'] << {
+          url: 'all_neighbors_system_dashboard/warehouse_reports/all_neighbors_system_dashboards',
+          name: 'All Neighbors System Dashboard',
+          description: 'All Neighbors System Dashboard',
```

Comment:
We should have a better description, maybe something about how this is Dallas specific?